test/unit/gen/certificate.go

```diff
@@ -110,6 +110,13 @@ func SetCertificateSecretName(secretName string) CertificateModifier {
 	}
 }
 
+func SetCertificateSecretTemplate(annotations, labels map[string]string) CertificateModifier {
+	return func(crt *v1.Certificate) {
+		crt.Spec.SecretTemplate.Annotations = annotations
+		crt.Spec.SecretTemplate.Labels = labels
+	}
+}
+
```

Comment:
Please add a short documentation comment to this public function.

Reply:
Done.